vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -216,6 +221,16 @@ def parse_container_port_config(port_config, pod_id, container_name)
       }
     end
 
+    def parse_service_port_config(port_config, service_id)
+      {
+        :ems_ref     => "#{service_id}_#{port_config["port"]}_#{port_config["targetPort"]}",
+        :name        => port_config["name"],
+        :port        => port_config["port"],
+        :target_port => port_config["targetPort"],
+        :protocol    => port_config["protocol"]
```

Comment:
> I thought dot notation was the way to do this now. cc @abonas

typically yes, but in some cases it doesn't work due to recursiveopenstruct design.
actually, @zeari, since kubeclient was updated to its latest version, it's also configured to support dot notation when array is used as part of the object, not only hash. so worth to try now.